aws_backendless_chef_ha.yml

```diff
@@ -0,0 +1,954 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: >
+  AWS HA Chef Server 2.0 - Backendless Chef - Ubuntu Xenial Based
+  "Backendless" Chef utilizing AWS as Backend for PostgreSQL and Elasticsearch
+  Sets up the following:
+    Backendless Items
+    * Elasticsearch 2.3 Cluster
+    * PostgreSQL 9.5.4
+
+Parameters:
+
+  ## Domain Configuration
+  HostedZone:
+    Type: String
+    Default: domain.com
+    Description: must match a route53 hosted domain/zone
+
+  SSLCertificateARN:
+    Type: String
+    Default: 'arn:aws:iam::'
+    Description: SSL Certficate ARN for SSL Certficate
+
+  ## Bucket Configuration
+  ExistingBucketName:
+    Type: String
+    Default: ''
+    Description: Leave Empty to disable, only fill out if using an existing bucket
+
+  ## Re-Deploy Configuration
+  ExistingInstall:
+    Type: String
+    Default: 'true'
+    AllowedValues:
+      - 'true'
+      - 'false'
+    Description: Choose only if existing install (i.e.; Previous External DB and Existing S3 Bucket)
+
+  ## Chef Configuration
+  ChefSubdomain:
+    Type: String
+    Default: chef-a
+    AllowedValues:
+      - chef-a
+      - chef-b
+      - chef-test
+    Description: subdomain/prefix for chose hosted zone used for staging
+
+  ChefServerVersion:
+    Type: String
+    Default: "12.11.1"
+    AllowedValues:
+      - "12.11.1"
```

Comment:
Most of the versions are due to Xenial support, Reporting is limited by support with Backendless or Chef Backend (aka Elasticsearch setups). This version isn't even fully supported.